cyber/tools/cyber_monitor/cyber_topology_message.cc

```diff
@@ -222,8 +222,7 @@ void CyberTopologyMessage::Render(const Screen* s, int key) {
     case SecondColumnType::MessageFrameRatio:
       s->AddStr(col1_width_ + SecondColumnOffset, 0, Screen::WHITE_BLACK,
                 "FrameRatio");
-      col2_width_ = 10;
-      s->AddStr(col1_width_ + col2_width_ + 2 * SecondColumnOffset, 0,
+      s->AddStr(col1_width_ + 10 + 2 * SecondColumnOffset, 0,
```

Comment:
Hard code is not good. Plx make 10 be one constexpr variable just like "SecondColumnOffset"

Reply:
ok